recipe/meta.yaml

```diff
@@ -1,8 +1,8 @@
 {% set name = "oauthlib" %}
-{% set version = "3.2.1" %}
+{% set version = "3.2.2" %}
 {% set compress_type = "tar.gz" %}
```

Comment:
all those extra variables used only once, isn't that is over engineered, can we simplify it and remove compress_type, hash_type and make consistent with other feedstocks ?

Reply:
It is ok to simplify the recipe if the variables are only used once.

Reply:
Done